showpass/Showpass-E2E-tests
# TA-57 Verify a popular city can be changed

**Description:**
This script will be verifying that when a popular city is selected, it can be changed (or updated) by selecting another value from the drop-down list box
Steps:
- Navigate to 
- [Create Amazing Experiences](https://beta.showpass.com/) 
- Click on a popular city Calgary | Vancouver | Toronto
- Ensure the city can be changed by selecting another city through the right-hand drop-down list box

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-57

**How to test:**
Run the following test suite and ensure the tests pass.
cypress/e2e/home/popular-cities.cy.js

**Test execution results:**

<img width="917" alt="TA-57-all-tests-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/aa57fbfe-d171-4c93-8895-e58ed4596d88">
